### PR TITLE
Document oneOf(), anyOf() and allOf()

### LINKED ISSUE #84: Mention oneOf, anyOf & allOf in the README
These combinators were introduced in #82.

### PR DESCRIPTION
**Pull request by sharplet**
Had a go at matching the style of the README, but found it kind of challenging to be both terse AND clear!

Fixes #84.

### CODE REVIEW

**Comment by robrix** [comment]
I’ve never been accused of being particularly clear, so no worries :heart:

File: README.md [hunk 1]
```diff
@@ -57,6 +57,24 @@ See `Madness.playground` for some examples of parsing with Madness.
...
+	oneOf([x1, x2, x3])
+	```
+
+	tries a sequence of parsers until the first success, producing parses as `X`
+
+	```swift
+	anyOf(["x", "y", "z"])
+	```
+
+	tries to parse any of a set of literals in sequence, collecting each unique successful parse into an array until none match
```
Comment (robrix) [inline README.md]: Maybe mention “one each”?

Comment (robrix) [inline README.md]: Actually, “in sequence” says that. Hm.

**Comment by robrix** [comment]
Could you please end the lines with `.`s?

Just that and the above note re: `anyOf`.

File: README.md (hunk 1 above)
Comment (robrix) [inline README.md]: Well, whatever you think, really!

Comment (sharplet) [inline README.md]: I like "one each" actually. Done.

**Comment by sharplet** [comment]
Ok that should be it.

**Comment by sharplet** [comment]
Where is my fingers-crossed emoji. :angry:

**Comment by robrix** [comment]
:ok_hand:

**Comment by robrix** [comment]
Very nice :sparkles: Thank you!

**Pull request merged**